client/src/components/Admin/OrganizationEdit.js

```diff
@@ -52,11 +52,20 @@ const validationSchema = Yup.object().shape({
   email: Yup.string().email("Invalid email address format"),
   hours: Yup.array().of(HourSchema),
   twitter: Yup.string()
+    .transform((value, originalValue) => (originalValue === "" ? null : value))
+    .matches(/^(?:@?[a-zA-Z0-9_]{1,15})$/, "Valid Twitter username required.")
+    .nullable(),
+
+  pinterest: Yup.string()
+    .transform((value, originalValue) =>
+      originalValue.trim() === "" ? null : value
+    )
     .matches(
-      /^https?:\/\/(www\.)?(twitter\.com|x\.com)\/.*/,
-      "Invalid URL, e.g. 'https://twitter.com/ or https://x.com/'"
+      /^[a-zA-Z0-9.]{3,15}$/,
```

Comment:
@hanapotski Thanks a lot for all your suggestions. 
@monicakrystal Great work on this PR, just wanted to share a few observations based on the [link](https://help.pinterest.com/en/article/edit-your-profile#:~:text=For%20example%2C%20Pinterest%20has%20the,between%20three%20and%2030%20characters) shared by Hannah:

1. The username allowed by the Pinterest policy can be 30 characters long and the current expression throws an error after 15.
2. Underscores are allowed by Pinterest whereas the dot is not.
3. Username can not be just made of the numbers however the above expression does not return an error if the username entered is just the numbers. I haven't figured out how to implement this yet, will share if I find anything. We can also discuss this during the meeting if this could be implemented later.

Thank you.